packages/web/src/javascripts/Components/NotesOptions/AddTagOption.tsx

```diff
@@ -92,14 +92,19 @@ const AddTagOption: FunctionComponent<Props> = ({
                   className={'ml-0.5 mr-1.5 h-7 w-7 text-2xl text-neutral lg:h-6 lg:w-6 lg:text-lg'}
                 />
               )}
-              <span
-                className={classNames(
-                  'overflow-hidden overflow-ellipsis whitespace-nowrap',
-                  isTagLinkedToSelectedItems(tag) ? 'font-bold' : '',
-                )}
-              >
-                {getTitleForLinkedTag(tag, application)?.longTitle}
-              </span>
+              <div>
+                <p className="overflow-hidden overflow-ellipsis whitespace-nowrap text-neutral">
```

Comment:
what does this look like if the tag doesn't have a hierarchy?

Reply:
It's just an empty <p>! I could add conditional rendering though to make it cleaner.

<img width="650" height="447" alt="Screenshot 2025-09-18 at 10 20 22 AM" src="https://github.com/user-attachments/assets/fdb042ce-81e1-4601-9b53-1789bc621708" />